samples/azure-iot-hub/CMakeLists.txt

```diff
@@ -7,13 +7,7 @@ if (CMAKE_C_COMPILER_ID STREQUAL "GNU")
   set (CMAKE_C_FLAGS "--std=c99 -g ${CMAKE_C_FLAGS}")
 endif()
 
-set(azure_IoT_Sdk_c $ENV{AZURE_IOT_SDK})
-include_directories(${azure_IoT_Sdk_c}/c-utility/inc
-                    ${azure_IoT_Sdk_c}/iothub_client/inc)
-link_directories(${azure_IoT_Sdk_c}/cmake/iotsdk_linux/serializer
-                  ${azure_IoT_Sdk_c}/cmake/iotsdk_linux/iothub_client
-                  ${azure_IoT_Sdk_c}/cmake/iotsdk_linux/c-utility
-                  ${azure_IoT_Sdk_c}/cmake/iotsdk_linux/umqtt)
+include_directories($ENV{SYSROOT}/usr/include/azureiot/inc)
 
 add_executable(azure-blink main.c)
 target_link_libraries(azure-blink wiringPi
```

Comment:
Yes, you can check the answer of this issue.

https://github.com/Azure/azure-iot-sdk-c/issues/4